### DISCUSSION TITLE
/cityselect/jinan.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#185] [Announcements]
# /cityselect/jinan.html

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/cityselect/jinan.html

**Comment by zonekoko** [comment]
二哥，能出一篇无锡的吗
